ChromeExtension/js/lastLinksViewModel.js

```diff
@@ -26,12 +28,17 @@ function lastLinksViewModel() {
             for (var i = 0; i < results.length; i++) {
                 self.resultsArray.push({
                     url: 'http://geni.us/' + results[i]['ShortUrlCode'],
-                    totalClicks: results[i]['TotalClicks']
+                    totalClicks: results[i]['TotalClicks'],
+                    editUrl: 'https://my.geni.us/links#!editlink=' + results[i]['ShortUrlCode']
                 });
             }
-        }
 
-        self.loadingOption('');
+            self.loadingOption('');
+            self.tableHeader('\
+                              <th>Links</th>\
```

Comment:
Does this HTML ever change? If not, why is it an observable? Or, if you are just showing/hiding it there are better ways to do that.